src/main/java/com/thealgorithms/datastructures/hashmap/hashing/MainCuckooHashing.java

```diff
@@ -0,0 +1,65 @@
+package com.thealgorithms.datastructures.hashmap.hashing;
+
+import java.util.Scanner;
+
+public class MainCuckooHashing {
+    public static void main(String[] args) {
```

Comment:
Can you write tests instead of console application?

Reply:
There is no rules about how to write tests in this repository, but it could be nice if you will use JUnit 5 with standard or AssertJ assertions.
You can use [this example](https://github.com/TheAlgorithms/Java/blob/master/src/test/java/com/thealgorithms/datastructures/lists/SkipListTest.java) as a reference

Reply:
Sure ill do that and reassess.

Reply:
Added MainCuckooHashingTest to project for testing purposes.

Please check and assure completeness.
Also if possible to assign issue to myself.